Scoop/Controller/Home/HomeViewController.swift

```diff
@@ -132,36 +132,36 @@ class HomeViewController: UIViewController {
     
     private func getRides() {
         // Populate table view with dummy data
-        let readesRide = Ride()
-        readesRide.organizer.name = "Reade"
-        readesRide.departureLocation = "Ithaca, NY"
-        readesRide.arrivalLocation = "Darien, CT"
-        readesRide.date = "Mar 3"
-        readesRide.isActive = true
-        
-        let annesRide = Ride()
-        annesRide.organizer.name = "Anne"
-        annesRide.departureLocation = "Ithaca, NY"
-        annesRide.arrivalLocation = "Orlando, FL"
-        annesRide.date = "Apr 10"
-        annesRide.isActive = true
-        
-        let karlsRide = Ride()
-        karlsRide.organizer.name = "Karl"
-        karlsRide.departureLocation = "Ithaca, NY"
-        karlsRide.arrivalLocation = "Pittsburgh, PA"
-        karlsRide.date = "May 13"
-        karlsRide.isActive = true
-        
-        let sarahsRide = Ride()
-        sarahsRide.organizer.name = "Sarah"
-        sarahsRide.departureLocation = "Ithaca, NY"
-        sarahsRide.arrivalLocation = "Montreal, CA"
-        sarahsRide.date = "May 20"
-        sarahsRide.isActive = true
-        
-        activeRides = [readesRide, annesRide]
-        pendingRides = [karlsRide, sarahsRide]
+//        let readesRide = Ride()
```

Comment:
Same here. If we ever comment code we should either 1. Just delete it instead or 2. if we plan on using it in the future document that with a comment and a "// TODO:"

Reply:
Did not realize I had to document directly in the code, I left a comment about this in the Changes Made section but I will be sure to also document this as well next time around.